Plugin.cs

```diff
@@ -21,18 +21,17 @@ public void Init(IPALogger logger, Config conf) {
 			Log = logger;
 			Log.Info("BetterSongSearch initialized.");
 			Settings.cfgInstance = PluginConfig.Instance = conf.Generated<PluginConfig>();
-
-			UI.Manager.Init();
-
-			SharedCoroutineStarter.Init();
-
 			new Harmony("Kinsi55.BeatSaber.BetterSongSearch").PatchAll(Assembly.GetExecutingAssembly());
 		}
 
 		[OnStart]
 		public void OnApplicationStart() {
+			BS_Utils.Utilities.BSEvents.lateMenuSceneLoadedFresh += LateMenuSceneLoadedFresh;
```

Comment:
Why is this necessary? I'd rather not depend on BS Utils, especially if its just for this

Reply:
Not really necessary, I'm only using it to get the lateMenuSceneLoadedFresh event, if you got a better way 👍 
Some BSML menu initialization can't be done in Init or OnEnable anymore.

Reply:
This is now removed.